test/auto/injectorSpec.js

```diff
@@ -302,6 +302,26 @@ describe('injector', function() {
           expect(instance.aVal()).toEqual('a-value');
         });
 
+        if (/chrome/.test(navigator.userAgent)) {
```

Comment:
I generally agree, but there is not much we can do here unfortunately. There is more context in the comments (e.g. starting at https://github.com/angular/angular.js/pull/14533#issuecomment-215488679).